#ifndef PROTOBUF_H
#define PROTOBUF_H

#include "createVote.pb.h"
#include "createVoteResponse.pb.h"
#include "getAllVotes.pb.h"
#include "getAllVotesResponse.pb.h"
#include "getVoteResult.pb.h"
#include "getVoteResultByOneJudge.pb.h"
#include "getVoteResultByOneJudgeResponse.pb.h"
#include "loginRequest.pb.h"
#include "loginResponse.pb.h"
#include "vote.pb.h"
#include "voteResponse.pb.h"

#endif